utils/markdown/renderer.tsx

```diff
@@ -56,99 +58,105 @@ export default class Renderer extends marked.Renderer {
+            );
             output = TextFormatting.replaceTokens(output, tokens);
         }
 
-        return (
-            '<span class="codespan__pre-wrap">' +
-                '<code>' +
-                    output +
-                '</code>' +
-            '</span>'
-        );
+        return oneLineTrim`<span class="codespan__pre-wrap">
+                <code>
+                    ${output}
+                </code>
+            </span>`;
     }
 
-    br() {
+    public br() {
         if (this.formattingOptions.singleline) {
             return ' ';
         }
 
         return super.br();
     }
 
-    image(href, title, text) {
+    public image(href: string, title: string, text: string) {
         let src = href;
-        let dimensions = [];
+        let dimensions: string[] = [];
         const parts = href.split(' ');
         if (parts.length > 1) {
             const lastPart = parts.pop();
             src = parts.join(' ');
-            if (lastPart[0] === '=') {
+            if (lastPart && lastPart[0] === '=') {
                 dimensions = lastPart.substr(1).split('x');
                 if (dimensions.length === 2 && dimensions[1] === '') {
                     dimensions[1] = 'auto';
                 }
             }
         }
         src = PostUtils.getImageSrc(src, this.formattingOptions.proxyImages);
-        let out = '<img src="' + src + '" alt="' + text + '"';
+        let out = `<img src="${src}" alt="${text}"`;
         if (title) {
-            out += ' title="' + title + '"';
+            out += ` title="${title}"`;
         }
         if (dimensions.length > 0) {
-            out += ' width="' + dimensions[0] + '"';
+            out += ` width="${dimensions[0]}"`;
         }
         if (dimensions.length > 1) {
-            out += ' height="' + dimensions[1] + '"';
+            out += ` height="${dimensions[1]}"`;
         }
         out += ' class="markdown-inline-img"';
         out += this.options.xhtml ? '/>' : '>';
         return out;
     }
 
-    heading(text, level) {
+    public heading(text: string, level: number) {
         return `<h${level} class="markdown__heading">${text}</h${level}>`;
     }
 
-    link(href, title, text, isUrl) {
+    public link(href: string, title: string, text: string) {
         let outHref = href;
 
         if (!href.startsWith('/')) {
             const scheme = getScheme(href);
             if (!scheme) {
                 outHref = `http://${outHref}`;
-            } else if (isUrl && this.formattingOptions.autolinkedUrlSchemes) {
-                const isValidUrl = this.formattingOptions.autolinkedUrlSchemes.indexOf(scheme.toLowerCase()) !== -1;
+            // eslint-disable-next-line prefer-rest-params
+            } else if (arguments[3] && this.formattingOptions.autolinkedUrlSchemes) {
+                const isValidUrl =
+          this.formattingOptions.autolinkedUrlSchemes.indexOf(
+              scheme.toLowerCase()
+          ) !== -1;
```

Comment:
Can you fix the indentation here? I'd also be a fan of keeping this all on one line as well. We don't have any particular line length rules here, but we generally aim around 120 characters to avoid unnecessary wrapping.